docs/howto/ejb-remote-security/index.adoc

```diff
@@ -0,0 +1,57 @@
+
+[#ejb-remote-security]
+
+= Enabling EJB-Remote Security
+
+To enable security for remote ejb invocation you need to add the `management` fraction
+and re-configure the `Application Realm` to enforce authentication.
+
+----
+<dependency>
+  <groupId>org.wildfly.swarm</groupId>
+  <artifactId>management</artifactId>
+</dependency>
+----
+
+----
+swarm:
+  management:
+    security-realms:
+      ApplicationRealm:
+        in-memory-authentication:
+          users:
+            bob:
+              password: tacos
+        in-memory-authorization:
+          users:
+            bob:
+              roles:
+              - admin
+  remoting:
+    http-connectors:
+      http-remoting-connector:
+        security-realm: ApplicationRealm
+----
+
+The ejb client needs to be configured to use credentials on the invocation as well:
```

Comment:
"ejb client" -> "EJB client"

Should also mention this file is called `wildfly-config.xml`.